internal/resources/azure/app_service_plan.go

```diff
@@ -0,0 +1,114 @@
+package azure
+
+import (
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+
+	"fmt"
+	"strings"
+
+	"github.com/shopspring/decimal"
+)
+
+type AppServicePlan struct {
+	Address     string
+	SKUSize     string
+	SKUCapacity int64
+	Kind        string
+	Region      string
+}
+
+var AppServicePlanUsageSchema = []*schema.UsageItem{}
+
+func (r *AppServicePlan) PopulateUsage(u *schema.UsageData) {
+	resources.PopulateArgsWithUsage(r, u)
+}
+
+func (r *AppServicePlan) BuildResource() *schema.Resource {
+	sku := r.SKUSize
+	skuRefactor := ""
+	os := "windows"
+	var capacity int64 = 1
+	if r.SKUCapacity > 0 {
+		capacity = r.SKUCapacity
+	}
+	productName := "Standard Plan"
+
+	if len(r.SKUSize) < 2 || strings.ToLower(r.SKUSize[:2]) == "ep" {
+		return &schema.Resource{
+			Name:      r.Address,
+			IsSkipped: true,
+			NoPrice:   true, UsageSchema: AppServicePlanUsageSchema,
+		}
+	}
+
+	switch strings.ToLower(sku[2:]) {
+	case "v1":
+		skuRefactor = sku[:2]
+		productName = "Premium Plan"
```

Comment:
```suggestion
	switch strings.ToLower(r.SKUSize[2:]) {
	case "v1":
		sku = r.SKUSize[:2]
		productName = "Premium Plan"
```

as per above if you agree. We can also update all the below cases to match.

Reply:
Fixed